docs/api/immutability-helpers.md

```diff
@@ -58,10 +58,13 @@ let state = {
 let planets = get(state, 'planets')
 console.log(planets) // { earth, mars }
 
-// Or an array of keys to access a deeply nested value
+// an array of keys to access a deeply nested value
```

Comment:
Hmmm. This is inconsistent. What if we just capitalized the first character of every comment in the file?